src/main/scala/uk/gov/nationalarchives/omega/api/services/Dispatcher.scala

```diff
@@ -147,7 +149,8 @@ class Dispatcher(
   ): IO[Unit] =
     businessResult match {
       case Right(businessResult) =>
-        localProducer.send(businessResult.content, requestMessage)
+        IO.println("Want to send a message..") *>
+          localProducer.send(businessResult.content, requestMessage)
```

Comment:
could this removed or replaced with `logger`?

Reply:
Removed. This was just for debugging.